src/components/gui/gui.jsx

```diff
@@ -6,8 +6,10 @@ const GreenFlag = require('../../containers/green-flag.jsx');
 const TargetPane = require('../../containers/target-pane.jsx');
 const Stage = require('../../containers/stage.jsx');
 const StopAll = require('../../containers/stop-all.jsx');
+const MenuBar = require('../menu-bar/menu-bar.jsx');
```

Comment:
I think there are a lot more camels in this PR because of the number of new (hyphenated) class names being used, which are converted to camel case by the css loader (+ css-modules). Other than that, I think this PR is consistent with how we've been doing case here and in other repos.